### ISSUE TITLE
Provision vagrant with a seeded environment

### ISSUE THREAD

**Issue by atticoos** [SandersForPresident/WordPress#52]
When provisioning vagrant it should seed a ready-to-go WP instance

**Comment by gileswells** [comment]
You're basically saying at the initial provision install wordpress, setup the database, and do everything that's needed to just be able to hit the site and get rolling on it right away? I might be able to spend a couple hours doing that if there is a desire for this.

**Comment by gileswells** [comment]
Ok well I should have checked the PR's first...

**Comment by atticoos** [comment]
Yeah as you saw I have some progress in #53, feel free to branch off https://github.com/ajwhite/WordPress/tree/provisioning and PR into my progress branch
